table/table.go

```diff
@@ -548,15 +543,10 @@ func (t *Table) blockCacheKey(idx int) []byte {
 	return buf
 }
 
-// blockOffsetsCacheKey returns the cache key for block offsets.
-func (t *Table) blockOffsetsCacheKey() []byte {
-	y.AssertTrue(t.id < math.MaxUint32)
-	buf := make([]byte, 6)
-	buf[0] = 'b'
-	buf[1] = 'o'
```

Comment:
Both ways it will be fine, right? Was there any reasoning behind having this earlier?

Reply:
I think it was there so that there is no conflict in the cache keys (since we had a mixed cache for blocks and indices before). But now we have separated the caches so conflict shouldn't arise.

Reply:
Earlier all the items, index and block were in the same cache and that's why there is this comment in BFCacheKey() function.

> 	// Without the "bf" prefix, we will have conflict with the blockCacheKey.

The bloom filter cache was added in https://github.com/dgraph-io/badger/pull/1260 but we didn't fix the old cache keys.

With this PR
Block Cache => Blocks [key: tableID+blockID]
Index Cache => BlockOffets[key: tableID] and BF[Key:"bf"+tableID]

@NamanJain8 @ahsanbarkati